appveyor.yml

```diff
@@ -13,22 +13,32 @@ environment:
     BATDIR: ci\appveyor\win32
     CROSS_VERSION: 1
 
-  # 32 julia latest Python-35
-  - JULIA_URL: "https://julialangnightlies-s3.julialang.org/bin/winnt/x86/julia-latest-win32.exe"
+  # 32 julia-1.0 Python-35
+  - JULIA_URL: "https://julialang-s3.julialang.org/bin/winnt/x86/1.0/julia-1.0-latest-win32.exe"
```

Comment:
Testing with Julia nightly yields some bizarre failures (see below).  The same code works with Julia 1.0.  I suggest to test with stable release.

From Python 3.5:


> `ErrorException("ccall: could not find function PyString_AsStringAndSize in library C:\projects\pyjulia\.tox\py\Scripts\python35")`
> --- https://ci.appveyor.com/project/Keno/pyjulia/build/1.0.175/job/wwr318to7jr1b5h6#L433

From Python 2.7:

> ```
>     def test_call_julia_function_with_python_args(self):
>         self.assertEqual(['A', 'B', 'C'],
>                          list(julia.map(julia.uppercase,
> >                                       array.array('u', [u'a', u'b', u'c']))))
> E       RuntimeError: Julia exception: MethodError(uppercase, (PyObject u'a',), 0x000061ca)
> ```
> --- https://ci.appveyor.com/project/Keno/pyjulia/build/1.0.175/job/wwr318to7jr1b5h6#L547